File: src/test/java/solutions/n200_number_of_islands/SolutionTest.java
package solutions.n200_number_of_islands;

import org.junit.Assert;
import org.junit.Test;

/**
 * @author : Sobornov Vladimir
 * @since : 02.12.2022
 **/
public class SolutionTest {
    char[][] testGrid1 = {{'1','1','1','1','0'}, {'1','1','0','1','0'}, {'1','1','0','0','0'}, {'0','0','0','0','0'}};
    char[][] testGrid2 = {{'1','1','0','0','0'}, {'1','1','0','0','0'}, {'0','0','1','0','0'}, {'0','0','0','1','1'}};

    @Test
    public void numIslandsTest1() {
        Assert.assertEquals(1, Solution.numIslands(testGrid1));
    }

    @Test
    public void numIslandsTes2() {
        Assert.assertEquals(3, Solution.numIslands(testGrid2));
    }
}

File: src/test/kotlin/solutions/n69_sqrtx/SolutionTest.kt
package solutions.n69_sqrtx

import org.junit.Assert
import org.junit.Test

class SolutionTest {
    val solution = Solution()

    @Test
    fun mySqrtTest1() {
        Assert.assertEquals(2, solution.mySqrt(4))
    }

    @Test
    fun mySqrtTest2() {
        Assert.assertEquals(2, solution.mySqrt(7))
    }

    @Test
    fun mySqrtTest3() {
        Assert.assertEquals(3, solution.mySqrt(15))
    }
}